test/extensions/INTEL/SPV_INTEL_joint_matrix/cooperative_matrix_apply.ll

```diff
@@ -0,0 +1,149 @@
+;; compiled from joint_matrix_apply_bf16.cpp from intel/llvm with some modifications
+
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv %t.bc --spirv-ext=+SPV_KHR_cooperative_matrix,+SPV_INTEL_joint_matrix -o %t.spv
```

Comment:
also add the case where only joint matrix extension is enabled

Reply:
Do you have in mind, that joint_matrix extension should implicitly enable cooperative matrix? I'm not expecting to see results other then SPV_KHR_cooperative_matrix/cooperative_matrix.ll would generate :)

Reply:
ah, alright then :)